mkdocs.yml

```diff
@@ -34,28 +35,33 @@ nav:
 
   - Home: index.md
   - Getting Started: getting_started.md
-  - Bill of Materials: bill_of_materials.md
-  - Hardware Setup: hardware_setup.md
-  - Software Setup: 
-      - Using SD Card Image: software_setup/sd_card.md
+  - Hardware:
+      - DIY open-source JetBot:
+          - Bill of Materials: bill_of_materials.md
+          - Hardware Setup: hardware_setup.md
+      - Third Party Kits: third_party_kits.md
+  - Software Setup:
+      - Wi-Fi setup: software_setup/wifi_setup.md
       - Using Docker Container: software_setup/docker.md
```

Comment:
To get linear flow for the software setup options, we may want to

1. Move wifi setup to Reference/Wi-Fi Setup
2. Add link from existing wifi setup section in "Using SD Card Image" to the detailed wifi-setup page
3. Add small wifi setup section in "Using Docker Container" which links to detailed wifi setup page

This way user can click either software setup option and follow a linear flow for getting started.

In the future, we may want to consider a larger reoganization.  For example,

* Getting Started
  * Step 1 - Get a JetBot
      - DIY Kit
         - Bill of Materials
         - Hardware Setup
      - Third Party Kits
  * Step 2 - Flash SD card image
        - JetPack SD card image
        - JetBot SD card image
  * Step 3 - Connect to WiFi
      * GUI
      * Command Line
  * Step 4 - Configure System
    * power mode
    * disable GUI
    * etc.
  * Step 5 - Install JetBot Software
    * Docker Containers
    * Native Installation

While not needed for this PR, I'm curious if this flow is intuitive in your eyes.

Best,
John

Reply:
I'm supportive of both the interim reorganization (moving the wifi doc) and the eventual large reorganization.
Committed the change and staged here.
https://tokk-nv.github.io/jetbot/master/